### PR TITLE
Connect to the local elements/liquid node

### PR DESCRIPTION
**Pull request by eckelj**

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
```
Comment (Cybnon) [inline .vscode/launch.json:1]: this folder should be added to .gitigonore

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
```
Comment (Cybnon) [inline .vscode/settings.json:1]: same as above

File: tests/test_liquid.py [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    resp = register_asset(
+        asset_id=asset_obj["asset_id"],
+        contract=json.loads(asset_obj["contract"]),
+        registration_endpoint=RDDL_ASSET_REG_ENDPOINT,
+    )
+    assert resp.status_code == 201
+
+
+#
+# curl https://assets-testnet.blockstream.info -H 'Content-Type: application/json' -d '{"asset_id": "d32421a6b81bb901d1e2bfabd46b924a67b24b3f2ce853209373eedaba99af73","contract":{"entity": {"domain": "lab.r3c.network"}, "issuer_pubkey": "02a1accfdfb6a3b57442ba3f30d2c1784ac12751a894ea556164d6c5c85bfa3122", "nft": {"token": "f75c0dad824b83bc59f9f6094f72d7657a3ace7cbd8143e3501c73198a2b0c4f", "ipld": "bafkreibebmkz4qkgw6qce56x33h7uf6vy5g7lstskbf2fj7rahvp2t23mm"}, "name": "Tspf4dsfS", "precision": 8, "ticker": "T4i9O", "version": 0}}'
```
Comment (Cybnon) [inline tests/test_liquid.py:21]: do we need to leave this uncommented code or could we delete ist?

File: x21e8/liquid.py [hunk 4]
```diff
@@ -1,71 +1,89 @@
...
+
+
+def create_contract(issue_request: IssuingRequest, nft_token: str, cid: str, pubkey: str):
+    contract = f'{{"entity":{{"domain":"{LIQUID_REGISTRATION_DOMAIN}"}}, "issuer_pubkey":"{pubkey}", "nft":{{"token":"{nft_token}", "ipld":"{cid}"}}, "name":"{issue_request.name}", "precision":{issue_request.precision}, "ticker":"{issue_request.ticker}", "version":{VERSION}}}'
+    contract = json.loads(contract)
+    contract_sorted = json.dumps(contract, sort_keys=True, separators=(",", ":"))
     contract_hash = hashlib.sha256(six.ensure_binary(contract_sorted)).hexdigest()
-    print(contract_hash)
     contract_hash_rev = "".join(reversed([contract_hash[i : i + 2] for i in range(0, len(contract_hash), 2)]))
+    print(contract)
```
Comment (Cybnon) [inline x21e8/liquid.py:57]: do we want to print this information or do we want to add some logging at some point? :)

File: x21e8/liquid.py [hunk 5]
```diff
@@ -1,71 +1,89 @@
...
     print(contract_hash_rev)
+    return contract, contract_hash_rev
 
-    raw_tx = rpc_connection.createrawtransaction([], [{"data": "00"}])
 
+def issue_tokens(issue_request: IssuingRequest, nft_token: str, cid: str):
+    rpc_connection = AuthServiceProxy(
+        get_liquid_auth_proxy_url(),
+    )
+    # rpc_connection.loadwallet()
```
Comment (Cybnon) [inline x21e8/liquid.py:67]: i gues this can be removed?

File: .vscode/launch.json (hunk 1 above)
Comment (eckelj) [reply .vscode/launch.json:1]: done

File: .vscode/settings.json (hunk 2 above)
Comment (eckelj) [reply .vscode/settings.json:1]: done

File: x21e8/liquid.py (hunk 5 above)
Comment (eckelj) [reply x21e8/liquid.py:67]: Put this to TODO. will be resolved in the next release.

**Review by Cybnon — APPROVED** [review]

**Pull request merged**